packages/components/Avatar/src/useAvatar.ts

```diff
@@ -161,6 +163,29 @@ export const removeTitlesFromName = (words: string[]): string[] => {
   return words.filter((word) => !titles.has(word));
 };
 
+/**
+ * A function which verifies whether words array contains multi-word title
+ *
+ * @param name
+ * @returns name without multi-word titles
+ */
+const removeMultiWordTitles = (name: string, idx: number = 0): string => {
+  if (!isMultiWordName || idx >= multiWordTitles.length) return name;
```

Comment:
Could we have a newline separating the conditional from the rest of the code for readability?

Reply:
fixed

Reply:
Our formatter removes new line if it's on the top of the function